jenkins/version-def.sh

```diff
@@ -59,6 +59,18 @@ case ${arch} in
 esac
 echo "cpu_arch is ${cpu_arch}"
 
+# PHASE_TYPE: CICD phase at which the script is called, to specify Spark shim versions.
+# regular: noSnapshots + snapshots
+# pre-release: noSnapshots only
+# arm-base: noSnapshots without cdh shims
+
+# Change PHASE_TYPE to 'arm-base' when building on arm CPU
+if [ "$cpu_arch" == "arm64" ]; then
+    PHASE_TYPE="arm-base"
+else
+    PHASE_TYPE=${PHASE_TYPE:-"regular"}
```

Comment:
I mean you could put this above cpu_arch detection.

And then
```
    aarch64|arm64)
        cpu_arch='arm64'
        PHASE_TYPE="arm-base"
        ;;
```

to update directly instead of another if statement